src/stumpy_core/rgba.cr

```diff
@@ -120,6 +120,32 @@ module StumpyCore
       to_rgb8
     end
 
+    def self.from_hsl(h, s, l)
+      h /= 360.0
+      s /= 100.0
+      l /= 100.0
+
```

Comment:
For `from_relative` to work, `r, g, b` must be floats
and I like initializing them unconditionally better

``` crystal
r = g = b = 1.0

if s != 0
...
```